distros/centos7/bin-x86/run_build.sh

```diff
@@ -58,7 +58,7 @@ fi
 
 echo "${REF}" >${OUTDIR}/REF
 
-CONFIGUREHASH=$(cat LUSTRE-VERSION-GEN $(find -name "*.m4" -o -name "*.ac") | md5sum | cut -f1 -d " ")
+CONFIGUREHASH=$(cat LUSTRE-VERSION-GEN $(find -name "*.m4" -o -name "*.ac") $(find ${KERNELDIR} -name .config) | md5sum | cut -f1 -d " ")
```

Comment:
well, if you don't trust the kernel you also need to look at the kernel version and other things. overall I just manually clean the cache every time I update the kernel - this provides the most insurance

Reply:
Good enough for me. I'll just drop this PR.